main.tf

```diff
@@ -0,0 +1,116 @@
+terraform {
+  required_providers {
```

Comment:
this can only exist once. Right now it is in both main.tf and versions.tf

Reply:
Yes, I got an error actually trying to do terraform init. I'm deleting in from main.tf